content/engine/install/sles.md

```diff
@@ -135,16 +135,16 @@ $ sudo zypper addrepo {{% param "download-url-base" %}}/docker-ce.repo
    ```console
    $ sudo zypper search -s --match-exact docker-ce | sort -r
  
-     v  | docker-ce | package | 3:24.0.0-3 | s390x | Docker CE Stable - s390x
-     v  | docker-ce | package | 3:23.0.6-3 | s390x | Docker CE Stable - s390x
+     v  | docker-ce | package | 3:25.0.0-3 | s390x | Docker CE Stable - s390x
+     v  | docker-ce | package | 3:23.0.7-3 | s390x | Docker CE Stable - s390x
```

Comment:
nit

```suggestion
     v  | docker-ce | package | 3:24.0.7-3 | s390x | Docker CE Stable - s390x
```

Reply:
doh! that's what you get when just manually updating those. Fixed 👍